### DISCUSSION TITLE
[2023/05/30] Frontend Night

### DISCUSSION THREAD

**Discussion by jjenko** [wantedly/frontend-night#259] [General]
## Policy

- 資料の準備をがんばらない
- 参加・途中退席は自由
- スキップしない

話したい・聞きたいネタを書いてく ✏️

ハッシュタグ [#frontend_night](https://twitter.com/hashtag/frontend_night?src=hash)

Prev: https://github.com/wantedly/frontend-night/discussions/258 

 _Created from https://github.com/wantedly/frontend-night/discussions/179 by [issue-creator](https://github.com/rerost/issue-creator)_

**Comment by s-kawabe** [comment]
# デザインシステム事例をながめてみる

(個人的にNotionにメモってあったのをwantedly/design-systemに投下した)

https://github.com/wantedly/design-system/issues/6

**Reply by s-kawabe** [reply]
WantedlyにおけるGraphic Standardのようなものはデザインシステムに組み込まれているところがおおい？

**Reply by s-kawabe** [reply]
- この中だとWantedlyと比較して、Spindleがやや世界観似てるのかなと思った
- デジタル庁のは一番わかりやすい
- Apronのこれ、うちでもやりたいかも？
<img width="590" alt="image" src="https://github.com/wantedly/frontend-night/assets/65433193/02192e2a-cf22-437f-b380-8cdd3c7f7d7e">
